pkg/semanticrelease/semantic-release.go

```diff
@@ -190,8 +191,21 @@ func (s *SemanticRelease) GetChangelog(releaseVersion *shared.ReleaseVersion) (*
 }
 
 // WriteChangeLog writes changelog content to the given file
-func (s *SemanticRelease) WriteChangeLog(changelogContent, file string) error {
-	return ioutil.WriteFile(file, []byte(changelogContent), 0644)
+func (s *SemanticRelease) WriteChangeLog(changelogContent, file string, overwrite bool) error {
+	newContent := make([]byte, 0)
+	newContent = append(newContent, []byte(changelogContent)...)
+
+	// only prepend the new changelog content if the the given file already exists
+	_, err := os.Stat(file)
+	if !overwrite && err == nil {
+		content, err := os.ReadFile(file)
```

Comment:
Instead of reading the file, you can open the file in append mode. 
Like:

```
f, err := os.OpenFile(filename, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0600)
if err != nil {
    panic(err)
}

defer f.Close()

if _, err = f.WriteString(text); err != nil {
    panic(err)
}
```

Reply:
With append, the changelog would be added at the end right? But I think that we should  prepend the latest changelog to a given file.

Reply:
Yeah, that is right.